apps/openchallenges/mcp-server/src/main/resources/application.yml

```diff
@@ -1,6 +1,5 @@
-openchallenges-mcp-server:
-  welcomeMessage: 'Welcome to the OpenChallenges MCP Server - dev mode'
-  apiBaseUrl: http://openchallenges-api-gateway:8082/api/v1
+welcome.message: 'Welcome to the OpenChallenges MCP Server - dev mode'
+api.base.url: http://openchallenges-api-gateway:8082/api/v1
```

Comment:
These two properties should be grouped like done previously.
Keep the properties organized as the other properties in this file.

Preferred:

```yaml
openchallenges:
    mcp-server:
      welcome-message: ''
      api-base-url: ''
```

Reply:
@tschaffter  this is not the latest codebase, current approach is using camel-cased.

Reply:
If I remember well, both camel and kebab case can be used in `application.yml`. I suggest to use kebab case because we use kebab case for other (Spring) properties in this file. Currently we don't a convention across projects in the monorepo, it would be nice to have one in the future. For now, we just need to be consistent within each `application.yml` file.

Reply:
sure, I can update to have consistent casing with other java projects.